ProjectLighthouse/Levels/Categories/LuckyDipCategory.cs

```diff
@@ -0,0 +1,25 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using LBPUnion.ProjectLighthouse.Extensions;
+using LBPUnion.ProjectLighthouse.PlayerData;
+using Microsoft.EntityFrameworkCore;
+
+namespace LBPUnion.ProjectLighthouse.Levels.Categories;
+
+public class LuckyDipCategory : Category
+{
+    public override string Name { get; set; } = "Lucky Dip";
+    public override string Description { get; set; } = "Randomized uploaded content";
+    public override string IconHash { get; set; } = "g820605";
+    public override string Endpoint { get; set; } = "luckydip";
```

Comment:
```suggestion
    public override string Endpoint { get; set; } = "lbp2luckydip";
```
This needs to match the endpoint name in SlotsController.cs which is `[HttpGet("slots/lbp2luckydip")]`
Either you need to add another `[HttpGet("slots/luckydip")]` or change this back to lbp2luckydip

Reply:
I fixed it :)